typescript/sdk/src/metadata/chainMetadataTypes.ts

```diff
@@ -98,6 +95,7 @@ export const ChainMetadataSchemaObject = z.object({
       name: z.string(),
       symbol: z.string(),
       decimals: ZUint.lt(256),
+      denom: z.string().optional(),
```

Comment:
I hadn't considered reusing symbol but I think it may fail in certain cases `utia` vs `tia`.
Per our chat I'll add a refinement enforcement for this.